psi4/driver/driver_cbs.py

```diff
@@ -1257,77 +1783,97 @@ def cbs(func, label, **kwargs):
+        :py:func:`~psi4.driver.driver_cbs.scf_xtpl_helgaker_2` if two valid basis
+        sets present in ``scf_basis``, and :py:func:`~psi4.driver.driver_cbs.xtpl_highest_1` 
+        otherwise.
 
         .. hlist::
            :columns: 1
 
            * :py:func:`~psi4.driver.driver_cbs.xtpl_highest_1`
-           * :py:func:`~psi4.driver.driver_cbs.scf_xtpl_helgaker_3`
+           * :py:func:`~psi4.driver.driver_cbs.xtpl_exponential_2` 
+           * :py:func:`~psi4.driver.driver_cbs.xtpl_power_2` 
+           * :py:func:`~psi4.driver.driver_cbs.xtpl_expsqrt_2` 
            * :py:func:`~psi4.driver.driver_cbs.scf_xtpl_helgaker_2`
            * :py:func:`~psi4.driver.driver_cbs.scf_xtpl_truhlar_2`
            * :py:func:`~psi4.driver.driver_cbs.scf_xtpl_karton_2`
+           * :py:func:`~psi4.driver.driver_cbs.xtpl_exponential_3`
+           * :py:func:`~psi4.driver.driver_cbs.scf_xtpl_helgaker_3`
 
     :type corl_scheme: Callable
     :param corl_scheme: |dl| ``xtpl_highest_1`` |dr| || ``corl_xtpl_helgaker_2`` || etc.
 
-        Indicates the basis set extrapolation scheme to be applied to the correlation energy.
-        Defaults to :py:func:`~psi4.driver.driver_cbs.corl_xtpl_helgaker_2` if two valid basis sets
-        present in ``corl_basis`` and :py:func:`~psi4.driver.driver_cbs.xtpl_highest_1` otherwise.
+        Indicates the basis set extrapolation scheme to be applied to the 
+        correlation energy. Defaults to :py:func:`~psi4.driver.driver_cbs.corl_xtpl_helgaker_2` 
+        if two valid basis sets present in ``corl_basis`` and 
+        :py:func:`~psi4.driver.driver_cbs.xtpl_highest_1` otherwise.
 
         .. hlist::
            :columns: 1
 
            * :py:func:`~psi4.driver.driver_cbs.xtpl_highest_1`
+           * :py:func:`~psi4.driver.driver_cbs.xtpl_power_2`
            * :py:func:`~psi4.driver.driver_cbs.corl_xtpl_helgaker_2`
 
     :type delta_scheme: Callable
     :param delta_scheme: |dl| ``xtpl_highest_1`` |dr| || ``corl_xtpl_helgaker_2`` || etc.
 
-        Indicates the basis set extrapolation scheme to be applied to the delta correction
-        to the correlation energy.
-        Defaults to :py:func:`~psi4.driver.driver_cbs.corl_xtpl_helgaker_2` if two valid basis sets
-        present in ``delta_basis`` and :py:func:`~psi4.driver.driver_cbs.xtpl_highest_1` otherwise.
+        Indicates the basis set extrapolation scheme to be applied to the delta 
+        correction to the correlation energy. Defaults to 
+        :py:func:`~psi4.driver.driver_cbs.corl_xtpl_helgaker_2` if two valid basis 
+        sets present in ``delta_basis`` and :py:func:`~psi4.driver.driver_cbs.xtpl_highest_1` 
+        otherwise.
 
         .. hlist::
            :columns: 1
 
            * :py:func:`~psi4.driver.driver_cbs.xtpl_highest_1`
+           * :py:func:`~psi4.driver.driver_cbs.xtpl_power_2` 
            * :py:func:`~psi4.driver.driver_cbs.corl_xtpl_helgaker_2`
 
     :type delta2_scheme: Callable
```

Comment:
Wait, why do we need exactly two, and what have these to do with basis set extrapolation? My go-to example is a relativistic correction, frozen core correction, and (Q) correction. That would be three corrections.

Reply:
If you're just doing "corrections", you're not doing basis set extrapolation - your stages would be simple differences of `xtpl_highest_1`. If you want "corrections" using extrapolated energies, you need either the `_2` or `_3` method, depending on the BSE function you want to use.